relayer/src/ethereum_checkpoints/tests/mod.rs

```diff
@@ -117,43 +179,38 @@ async fn init(network: Network) -> Result<()> {
         update: sync_update,
     };
 
-    let client = GearApi::dev().await?;
-    let mut listener = client.subscribe().await?;
+    let client = NodeClient::new().await?;
+    let program_id = {
+        let lock = client.lock().await;
 
-    let program_id = upload_program(&client, &mut listener, init).await?;
+        let mut listener = lock.client.subscribe().await?;
+
+        upload_program(lock.client, &mut listener, init).await?
+    };
 
     println!("program_id = {:?}", hex::encode(program_id));
 
     Ok(())
 }
 
-#[ignore]
-#[tokio::test]
-async fn init_sepolia() -> Result<()> {
-    init(Network::Sepolia).await
-}
-
-#[ignore]
 #[tokio::test]
 async fn init_holesky() -> Result<()> {
-    init(Network::Holesky).await
+    init(Network::Holesky, RPC_URL.into()).await
 }
 
-#[ignore]
 #[tokio::test]
 async fn init_mainnet() -> Result<()> {
-    init(Network::Mainnet).await
+    init(Network::Mainnet, "https://www.lightclientdata.org".into()).await
 }
 
-#[ignore]
 #[tokio::test]
-async fn init_and_updating() -> Result<()> {
-    let beacon_client = BeaconClient::new(RPC_URL.to_string(), None)
+async fn replay_back_and_updating() -> Result<()> {
```

Comment:
So this test is expected to run ~15 minutes I suppose? (I've checked that CI time is 20 min. for tests now)